web/src/hotkey.js

```diff
@@ -402,8 +402,8 @@ function handle_popover_events(event_name) {
 
 // Returns true if we handled it, false if the browser should.
 export function process_enter_key(e) {
-    if ($(".dropdown.open").length && $(e.target).attr("role") === "menuitem") {
-        // on #gear-menu li a[tabindex] elements, force a click and prevent default.
+    if ($(".dropdown.open, .dropup.open").length > 0 && $(e.target).attr("role") === "menuitem") {
+        // on dropdown menu elements, force a click and prevent default.
```

Comment:
Could this change be an independent prep commit? It seems like a general change to out hotkey handling of `enter` in dropup menus, not something specific to this feature, and it's very helpful to have systematic changes like that be their own commits.

Reply:
Thanks, done!